spark/common/src/test/scala/org/apache/sedona/sql/aggregateFunctionTestScala.scala

```diff
@@ -62,6 +67,26 @@ class aggregateFunctionTestScala extends TestBaseScala {
       assert(union.take(1)(0).get(0).asInstanceOf[Geometry].getArea == 10100)
     }
 
+    it("Measured ST_Union_aggr wall time") {
+      // number of random polygons to generate
+      val numPolygons = 1000
+      createPolygonDataFrame(numPolygons)
+
+      // cache the table to eliminate the time of table scan
+      sparkSession.sql("cache table geometry_table")
```

Comment:
Can you also unpersist this table at the end of the test case? Otherwise this will lead to memory leak.